moped-editor/src/components/FileUpload/FileUploadDialogSingle.js

```diff
@@ -161,22 +162,36 @@ const FileUploadDialogSingle = (props) => {
    */
   useEffect(() => {
     // Determine if the file is ready to be saved to DB
-    const saveDisabled =
-      fieldLength(fileName) === 0 ||
-      !Number.isInteger(fileType) ||
-      fieldLength(fileKey) === 0 ||
-      fileObject === null;
+    const saveDisabled = externalFile
+      ? fieldLength(fileName) === 0 ||
+        !Number.isInteger(fileType) ||
+        fieldLength(externalFileLink) === 0
+      : fieldLength(fileName) === 0 ||
+        !Number.isInteger(fileType) ||
+        fieldLength(fileKey) === 0 ||
+        fileObject === null;
```

Comment:
since saveDisabled is the opposite of fileReady logic wise, id appreciate a dummy check that I swapped the logic operators for the non external file upload.

Reply:
i double-checked this in the form, and it looks good to me!